values.yaml

```
@@ -32,6 +32,9 @@ podLabels: {}
 podSecurityContext: {}
   # fsGroup: 2000
 
+# workloadResourceAnnotations -- Set annotations on the Benthos Deployment
+workloadResourceAnnotations: {}
```

```suggestion
deployment:
  annotations: {}
```

I think this would be better placed under `deployment:`. `replicaCount`, `podAnnotations`, `podLabels` and `podSecurityContext` should also be moved under `deployment`

---

done. 

Also combined `terminationGracePeriodSeconds` with this same PR